test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -3257,15 +3257,15 @@ class TestArgmax(TestCase):
         (np.array(d[0], dtype=t), d[1])
         for d, t in (itertools.product(usg_data, (np.uint8,)))
     ]
-    darr = darr + [
+    darr = darr + [  # noqa: PIE794
```

Comment:
Lists are mutable in python ,aren't they? In that case, wouldn't it be better than adding `noqa:` suppression?
```suggestion
    darr += [
```

Reply:
Huh, that actually avoids the issue.